site/src/pages/WorkspaceSettingsPage/WorkspaceSettingsForm.tsx

```diff
@@ -78,7 +78,7 @@ export const WorkspaceSettingsForm: FC<WorkspaceSettingsFormProps> = ({
               workspace.allow_renames
                 ? form.values.name !== form.initialValues.name &&
                   "Depending on the template, renaming your workspace may be destructive"
-                : "Renaming your workspace can be destructive and has not been enabled for this deployment."
+                : "Renaming your workspace can be destructive and is disabled by the template."
```

Comment:
Is there something functional about this PR that requires this text change? or is this an extra change that adds some clarity here?

Reply:
I noticed it while referencing other bits of the app for design bits, and the old text isn't describing why it's disabled very accurately.